sites/main-site/src/content/blog/2025/retreat-2025.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: "nf-core core team retreat - September 2025"
+subtitle: "Updates from the nf-core core team retreat."
+pubDate: 2025-10-20T12:00:00+12:00
+headerImage: "/assets/images/blog/retreat-2025/team-wide.jpeg"
+headerImageAlt: "nf-core core team group photo taken during the 2025 retreat in Stockholm, Sweden."
+embedHeaderImage: true
+authors:
+    - "christopher-hakkaart"
+label:
+    - "community"
+---
+
+import { Image } from "astro:assets";
+import aifur from "@assets/images/blog/retreat-2025/aifur.jpeg";
+
+Between September 22nd and 26th, the [nf-core core team](https://nf-co.re/governance#core-team) gathered in Stockholm to tackle the big questions facing our growing community.
+Over four days, we worked through challenges around scaling, developer experience, and sustainability, and came away with a clear roadmap for the year ahead.
+
+# What did we talk about?
+
+We covered a wide range of topics during the retreat, from governance structure and hackathon strategy to documentation overhaul and technical infrastructure.
```

Comment:
since you removed the hackathon strategy from the main body, maybe consider to remove it from here, too.